### PR TITLE
Selectize.js for keypair

### PR DESCRIPTION
**Pull request by Joeper214**
### Objectives of this PR

Use Selectize.js when selecting AWS region for creating/editing a new infrastructure.

### CODE REVIEW

**Comment by pocke** [comment]
Please resolve conflict

**Comment by pocke** [comment]
Cool :cat:
default

**Comment by Joeper214** [comment]
Yeah. A simple implementation only. maybe later on, we can use this for other functionality enhancements.

File: bower.json [hunk 1]
```diff
@@ -22,6 +22,10 @@
     "jquery-ui": "~1.11.4",
     "vue": "~0.12.9",
     "lodash": "~3.9.3",
-    "zeroclipboard": "~2.2.0"
+    "zeroclipboard": "~2.2.0",
+    "selectize": "",
+    "microplugin": "",
+    "sifter": "",
+    "selectize-bootswatch": ""
```
Comment (pocke) [inline bower.json]: Can you fix version of libraries(e.g. `~1.0.0`)?

Comment (Joeper214) [inline bower.json]: Yeah sure, I forgot to include them. thanks!

Comment (Joeper214) [inline bower.json]: How about no release version? like in line 29. I have found problems in installing it. 



nover

Comment (pocke) [inline bower.json]: I think we specify git revision as version.
Please try  `"selectize-bootswatch": "5e9042f617"`

**Comment by Joeper214** [comment]
Done. Thanks!

**Comment by pocke** [comment]
:+1:

**Pull request merged**